docs/feature/002-vip-counter.md

```
@@ -0,0 +1,66 @@
+   如果存在，则删除: iptables -t nat -D vip.out.counter -i eth1 -s $vipInfo.Ip/32 -m comment --comment $vipInfo.VipUuid -j RETURN
+
+在setVipHandler的函数增加逻辑:
+遍历cmd.Vips, 调用前面的函数为每个vip删除iptables规则
+
+## 读取iptables规则计数
+
+添加一个函数: 通过iptables命令得到vip的流量统计:
+每个vip的统计值包含:
+vipUuid, vipIp, 入方向pkt计数，入方向octet计数，出方向pkt计数，出方向octet计数
+通过如下命令获取方向计数：
+> #  iptables-save -c | grep vip.in.counter
```

markdownlint-cli2 (0.18.1)

Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)